README.md

```diff
@@ -973,6 +972,61 @@ Learn more about class-transformer and how to handle more complex object constru
 This behaviour is enabled by default.
 If you want to disable it simply pass `useClassTransformer: false` to createExpressServer method.
 
+## Auto validating action params
+
+Sometimes parsing a json object into instance of some class is not enough. 
+E.g. `class-transformer` doesn't check whether the property's types are correct, so you can get runtime error if you rely on TypeScript type safe. Also you may want to validate the object to check e.g. whether the password string is long enough or entered e-mail is correct.
+
+It can be done easily thanks to integration with [class-transformer-validator][7] which internally depends on [class-validator][6]. So first of all, you have to install the modules:

```

Comment:
I would like to use it also in my other projects and not to be forced to download the whole `routing-controllers`.
If you are afraid because it's not your npm module, etc., I could lock the dependency version to 0.1.1 😉